kani-driver/src/metadata.rs

```diff
@@ -12,15 +12,14 @@ use std::fs::File;
 use std::io::{BufReader, BufWriter};
 
 use crate::session::KaniSession;
+use serde::Deserialize;
 
 fn generate_mock_harness() -> HarnessMetadata {
     HarnessMetadata {
         pretty_name: String::from("harness"),
         mangled_name: String::from("harness"),
         original_file: String::from("target_file.rs"),
-        original_start_line: 0,
-        original_end_line: 0,
-        unwind_value: None,
+        ..Default::default()
```

Comment:
I think it's valuable to have the ability to make a change to a type (e.g. add a field) then go see everywhere it breaks.

I try to get rid of places where we match on `_` in `kani-compiler` for the same reason. Then a change to a variant will cause a build error and point you to look at something.

It might be "busy work" to fix it half the time, but the "audit" is nice.

Reply:
I see your point. There are pros and cons here, since ideally this structure shouldn't be empty, I removed the default.